src/js/modal-dialog.js

```diff
@@ -493,17 +492,7 @@ class ModalDialog extends Component {
     const allChildren = this.el_.querySelectorAll('*');
 
     return Array.prototype.filter.call(allChildren, (child) => {
-      return ((child instanceof window.HTMLAnchorElement ||
-               child instanceof window.HTMLAreaElement) && child.hasAttribute('href')) ||
-             ((child instanceof window.HTMLInputElement ||
-               child instanceof window.HTMLSelectElement ||
-               child instanceof window.HTMLTextAreaElement ||
-               child instanceof window.HTMLButtonElement) && !child.hasAttribute('disabled')) ||
-             (child instanceof window.HTMLIFrameElement ||
-               child instanceof window.HTMLObjectElement ||
-               child instanceof window.HTMLEmbedElement) ||
-             (child.hasAttribute('tabindex') && child.getAttribute('tabindex') !== -1) ||
-             (child.hasAttribute('contenteditable'));
+      return Dom.isFocusable(child);
```

Comment:
I think `Dom.isFocusable` can just be passed directly without wrapping it in the arrow function.